### DISCUSSION TITLE
Bluetooth lock through Gateway

### DISCUSSION THREAD

**Discussion by phil-dickson** [make-all/tuya-local#906] [Q&A]
Hi. I'm looking to see if this will work for a Bluetooth lock. The readme says only WIFI devices are supports, but I saw someone say on a Reddit that now some Bluetooth devices are supports via gateways. I've looked up some of the issues. I did have local tuya and that is now removed. I've checked HA and the Gateway are on the same network. HA is on ethernet, and the tuya app is closed on my phone.

For the Device Id, IP Address and Local key, I'm selecting the values from the Gateway. Then in the Sub Device Id I'm selecting the unique Id of the lock. 

I'm getting this error come back: "Unable to connect to your device with those details. It could be an intermittent issue, or they may be incorrect."

My guess is my lock isn't supported, but I thought I'd ask the question in case I'm doing something stupid that isn't one of the above. Happy to provide any logs.

**Comment by make-all** [comment]
Unable to connect means either one of the connection details is incorrect, or the device is returning no data once connected.
If the device is returning data, it should get to the next step, even if it is an unsupported device.
